locales/en-US/app.ftl

```diff
@@ -102,6 +128,40 @@ ListOfPublishedProfiles--uploaded-profile-information-list =
        *[other] Manage these recordings
     }
 
+## ProfileDeleteButton
+
+# This string is used on the tooltip of the published profile links delete button in uploaded recordings page.
+# Variables:
+#   $smallProfileName (String) - Shortened name for the published Profile.
+ProfileDeleteButton--delete-button =
+    .label = Delete
+    .title = Click here to delete the profile { $smallProfileName }
```

Comment:
Ideally we should reuse the same strings as in:
https://github.com/firefox-devtools/profiler/blob/8fe95c6e6bd24a449c154c5ccb8019e9350ef7cb/locales/en-US/app.ftl#L80-L87

Because that's the exact same button, implemented differently.

@flodolo by chance do you have a suggestion to do this properly?

Reply:
You could reference the other message, but I'm not a fan of [message references](https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#message-references) in general.

Personally, I think it's OK to have the text repeated, then tools like Pontoon will show existing matches in the translation memory.